subtest's testcase.Cleanup conflicts with Go 1.14's new Cleanup

golang/go#32111 adds a cleanup function to the standard lib's testing types. This breaks `x/subtest` at compile time, like:

```
# gotest.tools/v3/x/subtest
../../../nobackup/gotip_home/gopath/pkg/mod/gotest.tools/v3@v3.0.0/x/subtest/context.go:63:10: cannot use tc (type *testcase) as type TestContext in argument to subtest:
	*testcase does not implement TestContext (wrong type for Cleanup method)
		have Cleanup()
		want Cleanup(func())
../../../nobackup/gotip_home/gopath/pkg/mod/gotest.tools/v3@v3.0.0/x/subtest/context.go:81:5: cannot use &testcase literal (type *testcase) as type TestContext in assignment:
	*testcase does not implement TestContext (wrong type for Cleanup method)
		have Cleanup()
		want Cleanup(func())
```

As `testcase` defines a `Cleanup` function of its own, conflicting with `testing.TB`'s definition. This type is not exported, and `Cleanup` does not appear to be used externally, only in `x/subtest`. Can this function be unexported or renamed to allow for this package to compile at tip?

--- a/x/subtest/context.go
+++ b/x/subtest/context.go
@@ -27,9 +27,9 @@ func (tc *testcase) Ctx() context.Context {
 	return tc.ctx
 }
 
-// Cleanup runs all cleanup functions. Functions are run in the opposite order
+// cleanup runs all cleanup functions. Functions are run in the opposite order
 // in which they were added. Cleanup is called automatically before Run exits.
-func (tc *testcase) Cleanup() {
+func (tc *testcase) cleanup() {
 	for _, f := range tc.cleanupFuncs {
 		// Defer all cleanup functions so they all run even if one calls
 		// t.FailNow() or panics. Deferring them also runs them in reverse order.
@@ -59,7 +59,7 @@ type parallel interface {
 func Run(t *testing.T, name string, subtest func(t TestContext)) bool {
 	return t.Run(name, func(t *testing.T) {
 		tc := &testcase{TB: t}
-		defer tc.Cleanup()
+		defer tc.cleanup()
 		subtest(tc)
 	})
 }

